.github/workflows/build.yml

```diff
@@ -69,5 +69,4 @@ jobs:
         run: |
           echo "//registry.npmjs.org/:_authToken=${NPM_AUTH_TOKEN}" > .npmrc
           yarn
-          yarn build
```

Comment:
Personally, I'm really glad for the use of documented lifecycles and features of `nmp`/`yarn` (such as `prepack` and `files`). This compounds the value of learning about those tools, and has the effect of making things more predictable across projects.